src/JustSaying.Tools/Commands/MoveCommand.cs

```diff
@@ -1,93 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Amazon;
-using Amazon.SQS;
-using Amazon.SQS.Model;
-using JustSaying.AwsTools;
-using JustSaying.AwsTools.MessageHandling;
-using Microsoft.Extensions.Logging;
-#pragma warning disable 618
-
-namespace JustSaying.Tools.Commands
-{
-    public class MoveCommand : ICommand
-    {
-        public MoveCommand(string sourceQueueName, string destinationQueueName, string region, int count)
-        {
-            SourceQueueName = sourceQueueName;
-            DestinationQueueName = destinationQueueName;
-            Region = region;
-            Count = count;
-        }
-
-        public string SourceQueueName { get; set; }
-        public string DestinationQueueName { get; set; }
-        public string Region { get; set; }
-        public int Count { get; set; }
-
-        public async Task<bool> ExecuteAsync()
-        {
-            Console.WriteLine($"Moving {Count} messages from {SourceQueueName} to {DestinationQueueName} in {Region}.");
-            using var loggerFactory = new LoggerFactory();
-
-            var config = new AmazonSQSConfig { RegionEndpoint = RegionEndpoint.GetBySystemName(Region) };
-            var client = new DefaultAwsClientFactory().GetSqsClient(config.RegionEndpoint);
-            var sourceQueue = new SqsQueueByName(config.RegionEndpoint, SourceQueueName, client, JustSayingConstants.DefaultHandlerRetryCount, loggerFactory);
-            var destinationQueue = new SqsQueueByName(config.RegionEndpoint, DestinationQueueName, client, JustSayingConstants.DefaultHandlerRetryCount, loggerFactory);
-
-            await EnsureQueueExistsAsync(sourceQueue).ConfigureAwait(false);
-            await EnsureQueueExistsAsync(destinationQueue).ConfigureAwait(false);
-
-            var messages = PopMessagesFromSourceQueue(sourceQueue);
-            var receiptHandles = messages.ToDictionary(m => m.MessageId, m => m.ReceiptHandle);
-
-            var sendResponse = destinationQueue.Client.SendMessageBatch(new SendMessageBatchRequest
```

Comment:
This Client property is private now as we no longer expose the IAmazonSqs, so I've removed this command as I don't think its used very much. We have other ways of handling retries.